ui/package.json

```diff
@@ -71,6 +71,7 @@
     "ember-cli-clipboard": "^0.13.0",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-deprecation-workflow": "^2.1.0",
+    "ember-cli-flash": "^3.0.0",
```

Comment:
Hm interesting -- I took to ember-clli-flash after asking about it in our frontend slack channel and being told flash messages were the norm here. I'm open to either, especially since notifications seem more recently updated.